test/Npgsql.Tests/MultipleHostsTests.cs

```diff
@@ -250,6 +249,7 @@ public async Task First_host_is_down()
     [TestCase("prefer-standby")]
     [TestCase("read-write")]
     [TestCase("read-only")]
+    // TODO: rewrite with DataSource?
```

Comment:
We can, but it seems important to at least have some coverage of TargetSessionAttributes as a connection string parameter ("legacy mode")